core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -127,59 +206,124 @@ SSLContext createSslContext() {
     }
 
     private KeyManager[] getKeyManagers() {
-        if (this.keyStorePath == null) {
+        if (this.keyStoreSource == null && this.trustStoreType != KeyStoreType.PKCS11) {
```

Comment:
Not sure why the condition is here, and it definitely shouldn't refer to `trustStoreType`.